crates/yarn_slinger/tests/dialogue_tests.rs

```diff
@@ -115,38 +114,42 @@ fn test_prepare_for_line() {
 
     let result = Compiler::new().read_file(path).compile().unwrap();
 
-    let mut dialogue = TestBase::new().with_compilation(result).dialogue;
-
-    let prepare_for_lines_was_called = Arc::new(RwLock::new(false));
-    let prepare_for_lines_was_called_clone = prepare_for_lines_was_called.clone();
-    dialogue.set_prepare_for_lines_handler(move |lines, _| {
-        // When the Dialogue realises it's about to run the Start
-        // node, it will tell us that it's about to run these two
-        // line IDs
-        assert_eq!(lines.len(), 2);
-        assert!(lines.contains(&"line:test1".into()));
-        assert!(lines.contains(&"line:test2".into()));
-
-        // Ensure that these asserts were actually called
-        *prepare_for_lines_was_called_clone.write().unwrap() = true;
-    });
-
-    dialogue.set_node_to_start();
-
-    assert!(*prepare_for_lines_was_called.read().unwrap());
+    let mut dialogue = TestBase::new()
+        .with_compilation(result)
+        .dialogue
+        .with_node_at_start();
+
+    let mut prepare_for_lines_was_called = false;
+
+    let events = dialogue.continue_().unwrap();
+    for event in events {
+        if let DialogueEvent::PrepareForLines(lines) = event {
+            // When the Dialogue realises it's about to run the Start
+            // node, it will tell us that it's about to run these two
```

Comment:
keep this comment on two lines

Reply:
[`e7a003d` (#103)](https://github.com/yarn-slinger/yarn-slinger/pull/103/commits/e7a003d1aecddeca2b08f7646ea39af34ea796ce)